src/components/Settings/Settings.js

```diff
@@ -71,9 +70,84 @@ const TabPaneGeneral = (store) => {
       </span>
     );
   });
+});
+
+const LayoutSettings = observer(({ store }) => {
+  return (
+    <>
+      <Checkbox
+        checked={store.settings.bottomSidePanel}
+        onChange={() => {
+          store.settings.toggleBottomSP();
+          setTimeout(triggerResizeEvent);
+        }}
+      >
+            Move sidepanel to the bottom
+      </Checkbox>
+
+      <br />
```

Comment:
rather than use `<br />`s can we wrap the checkboxes in `<Elem>` and potentially apply css as needed? maybe with a `<Block>` above instead of the `<>` so we can be more specific

Reply:
Fair point. I didn't want to touch this part as it's out of scope of the task, but can definitely do that.